database/src/main/java/com/firebase/ui/database/FirebaseArray.java

```diff
@@ -95,19 +96,27 @@ public void onChildMoved(DataSnapshot snapshot, String previousChildKey) {
     }
 
     public void onCancelled(DatabaseError firebaseError) {
-        // TODO: what do we do with this?
+        notifyCancelledListeners(databaseError);
```

Comment:
Thanks again! I'm working on another branch on my machine so I've been using the online file editor in Github. Yeah... It's tough without Intellij!